Beta-23/ilearnx
# Components

- Refactor person utils function
- Convert to name export in the person utils
- Test default export from utils
- Convert to inline default export
- Create default for senior person utils
- Install the validator module for email
- Test validator method to check for valid email
- Install react and react-dom
- Install babel core in order to integrate with webpack
- Configure the babel loader
